docroot/themes/contrib/civictheme/includes/banner.inc

```diff
@@ -176,6 +176,15 @@ function _civictheme_preprocess_civictheme_banner_node(&$variables) {
  */
 function _civictheme_preprocess_civictheme_banner_last_updated(&$variables, $node) {
   $last_updated = civictheme_get_field_value($node, 'field_c_n_show_last_updated');
+  $custom_updated = civictheme_get_field_value($node, 'field_c_n_custom_last_updated');
```

Comment:
@rebelmusic 
Please rationalise this to get the date in one place and render in one place instead of repeating rendering twice